code/modules/uplink/uplink_items/badass.dm

```diff
@@ -40,7 +40,7 @@
 			and services at lucrative prices. The briefcase also feels a little heavier to hold; it has been \
 			manufactured to pack a little bit more of a punch if your client needs some convincing."
 	item = /obj/item/storage/secure/briefcase/syndie
-	cost = 5
+	cost = 2
```

Comment:
`	cost = 3`
Best compromise I can make, but I want to keep an eye on this to see if it get abused to hell. Sound good?

Reply:
And still practice use of this money is low.

Reply:
But 3 is better than 5 so i'll just reduce it at some point if nothing like you described will happen.